### PR TITLE
adds ecommerceFlow

### PR DESCRIPTION
**Pull request by lucamattana**

### CODE REVIEW

File: e2e/Playmakers/front/spec/shop/ecommerceFlow.spec.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  });
+
+  test(" expect(page.url()).toContain(url + /products", async ({ page }) => {
+    console.log("test2", currentURL);
+    // expect(page.url()).toContain(url + "/products");
+  });
+
+  // test("add to cart", async ({ page }) => {});
+
+  // test("go to checkout", async ({ page }) => {});
```
Comment (LeandroOnline) [inline e2e/Playmakers/front/spec/shop/ecommerceFlow.spec.ts:34]: Need to delete useless comments

File: e2e/Playmakers/front/spec/shop/ecommerceFlow.spec.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      .click();
+
+    expect(page.url()).toContain(url + "/products");
+    currentURL = page.url();
+    console.log("test1", currentURL);
+  });
+
+  test(" expect(page.url()).toContain(url + /products", async ({ page }) => {
+    console.log("test2", currentURL);
+    // expect(page.url()).toContain(url + "/products");
```
Comment (LeandroOnline) [inline e2e/Playmakers/front/spec/shop/ecommerceFlow.spec.ts:29]: here also need to delete

**Review by LeandroOnline — CHANGES_REQUESTED** [review]
Hi luca, please see this before merging, thanks

File: e2e/Playmakers/front/spec/shop/ecommerceFlow.spec.ts (hunk 2 above)
Comment (lucamattana) [reply e2e/Playmakers/front/spec/shop/ecommerceFlow.spec.ts:29]: these commented sections need to stay, they are needed further down the development process

File: e2e/Playmakers/front/spec/shop/ecommerceFlow.spec.ts (hunk 1 above)
Comment (lucamattana) [reply e2e/Playmakers/front/spec/shop/ecommerceFlow.spec.ts:34]: these too

**Review by LeandroOnline — APPROVED** [review]

**Pull request merged**